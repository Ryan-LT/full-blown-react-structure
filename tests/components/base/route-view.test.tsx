import React from 'react';
import { RouterView } from '@components';
import { cleanup, render } from '../../renderTest';
import '@testing-library/jest-dom';

afterEach(cleanup);

describe('<RouteView />', () => {
  it('should take a snapshot', () => {
    const { asFragment } = render(<RouterView routes={[]} />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('should render without issue', () => {
    const component = render(<RouterView routes={[]} />);
    expect(component).toBeTruthy();
  });
});
