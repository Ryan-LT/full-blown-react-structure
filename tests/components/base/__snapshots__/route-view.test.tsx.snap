// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RouteView /> should take a snapshot 1`] = `<DocumentFragment />`;
